solo_study.md

```diff
@@ -48,16 +48,16 @@ Today you objectives are to put in practice the concepts you learned yesterday.
 
 *Note: Refer to this handy [Javascript Cheat sheet](http://overapi.com/javascript) if you need it*
 
-### Plan
+### Plan  http://jsbin.com/vetazuv/6/edit?js,console
 
-- [ ] Complete the last 5 tasks of [Objects Basics Tutorial](https://learn.javascript.info/object)
-  - [ ] Link to your solution here (jsbin, gist, or a js file in this repo)
-- [ ] Complete the last 5 tasks of [Arrays Tutorial](https://learn.javascript.info/array)
- - [ ] Link to your solution here
-- [ ] Complete the last 5 tasks of [Array Methods](https://learn.javascript.info/array-methods)
- - [ ] Link to your solution here
-- [ ] Complete the [Learn JS Basics tutorial](http://www.learn-js.org/en/Welcome)
-- [ ] Complete the [Learn JS Objects tutorial](http://www.learn-js.org/en/Object_Oriented_JavaScript)
+- [X] Complete the last 5 tasks of [Objects Basics Tutorial](https://learn.javascript.info/object)
+- [X ]   solution http://jsbin.com/vetazuv/6/edit?js,console or https://repl.it/GIOI/0
```

Comment:
```
// Multiply numeric properties by 2
let menu = {
  width: 200,
  height: 300,
  title: "My menu"
};

Object.values(menu).forEach(function(element) {
  if( Number.isInteger(element) === false){
    Object.values(element);
    return Object.values(element);
    
  }
    console.log(element*2);
```

This doesn't multiply element, it only multiplies in the logs but the actual value does not change. You'd want to console log the object instead of the element to check to see if it works